Simonyi/Non-Standard-Operations/MTCS/MTHexRot/MTHex-PXI-Controller-Reboot.rst

```diff
@@ -85,44 +85,23 @@ Restart Control System
     As the primary troubleshooting step to regain control, perform the restart of the control system if 
     the CSC/EUI is entirely unresponsive and unable to establish a connection with the control system.
 
-    #.  Before proceeding, make sure there are no active interlocks in the EUI and that no other EUI instance 
-        is running. For the MTCamHexapod, check if *runCamHexEui* is running by following commands in the terminal:
-
-        .. prompt:: 
-
-            ps -aux | grep runCamHexEui
-
-        If processes are already running, you may need to identify who is running them and ask permission 
-        to end one (or both) so you can run your own EUI session. If another *runCamHexEui* is running
-
-        .. prompt:: 
-
-            sudo kill -9 {pid}
+    Before proceeding, make sure there are no active interlocks in the EUI and that no other EUI instance 
+    is running. For the MTCamHexapod, check if *runCamHexEui* is running by following commands in the terminal:
 
+    .. prompt:: 
         
-        Replace *runCamHexEui* with *runM2HexEui* for the *MTM2Hexapod*.
-    
-    
-    #.  You can do the following to check the *MTCamHexapod* control system status in the terminal:
+        ps -aux | grep runCamHexEui
 
-        .. prompt::
+    If processes are already running, you may need to identify who is running them and ask permission 
+    to end one (or both) so you can run your own EUI session. If another *runCamHexEui* is running
 
-            /etc/init.d/cameraHexapod status
-        
-        It will tell you whether the control system is running or not. To stop it, do:
-
-         .. prompt::
-
-            /etc/init.d/cameraHexapod stop
+    .. prompt:: 
 
-        To start it do:
+        sudo kill -9 {pid}
 
-         .. prompt::
+        
+    Replace *runCamHexEui* with *runM2HexEui* for the *MTM2Hexapod*.
 
-            /etc/init.d/cameraHexapod start
-       
-        For *MTM2Hexapod*, replace in the command above *cameraHexapod* 
-        with *m2Hexapod*.
 
 
 #.  **Establish an SSH connection to the MTHexapod PXI**: Using the credentials and hostnames found 
```

Comment:
I think you should create a NOTE titled Checking "Ethercat connection"  and include  the following points from Te-Wei's message here as the first point BEFORE establishing connection - 
"If there is the connection issue, you could try:
ping  the controller first to see there is the internet response or not.
ssh to the controller. To check the EtherCAT is running correctly or not, do ethercat master should show the frame rate from the slaves. The value should be 200 Hz in hexapod and 4000 Hz for the rotator.
To confirm the system is running or not, you can do:
-> /etc/init.d/ethercat status (for EtherCAT).
-> /etc/init.d/hexapod status (for hexapod)
-> /etc/init.d/rotator status (for rotator). 

The `ethercat master` is the most straight-forward way to know the EtherCAT is running correctly or not. If it is not, the control system hangs up in an internal stage to wait until the EtherCAT is running correctly and you would not be able to use the GUI/CSC to connect to the control system at the moment. 
As a reference, the output of ethercat master would look like this:" <include the code block from his message >

Reply:
As I mentioned in a DM, I won't include the ethercat diagnosis, as the etherCat debug is beyond the scope for now. We might add it in a future release but for now, Te-Wei agreed we can leave it out for now.

Reply:
ok. One more comment --
1. can you add that this ssh tunnel has to be created from our local computer/laptop ? I saw many of us getting confused and peopln trying to do it from the hexrot machine. I think a clear explicit instruction is missing here. something like -- "open a terminal locally and create ssh tunnel .... "

Reply:
I don't think it matters from where you create the ssh tunnel to the PXI, locally or from the hexrot VM, as long as it´s in the WAP network.